install/kubernetes/cilium/charts/config/templates/configmap.yaml

```
@@ -165,10 +165,14 @@ data:
   bpf-ct-global-tcp-max: "{{ .Values.global.bpf.ctTcpMax }}"
   bpf-ct-global-any-max: "{{ .Values.global.bpf.ctAnyMax }}"
 
-  # bpf-nat-global-max specified the maximum number of entries in the BPF NAT
-  # table.
+  # bpf-nat-global-max specified the maximum number of entries in the
+  # BPF NAT table.
   bpf-nat-global-max: "{{ .Values.global.bpf.natMax }}"
 
+  # bpf-neigh-global-max specified the maximum number of entries in the
```

Nit: same here, probably C&P from above?

```suggestion
  # bpf-neigh-global-max specifies the maximum number of entries in the
```

---

Yeah, I can fix all three up at once, right now the other two (nat, policy) have the same.